zipkin/src/test/java/zipkin/TestObjects.java

```diff
@@ -78,6 +78,32 @@
           .build()
   ).stream().map(ApplyTimestampAndDuration::apply).collect(toList());
 
+
+  // this object simulates
+  public static final List<Span> TRACEWITHSAMEIDANDSAMEPARENTID = asList(
```

Comment:
please move this into a local variable in its only caller: DependencyLinkerTest.linksSpansShouldHandleNullSpans() also please don't concatenate words like this as it is not readable. Once a local variable use lower-camel